### ISSUE TITLE
Questions or statements in black cards in quotation marks

### ISSUE THREAD

**Issue by rfuehrer** [Lattyware/massivedecks#162]
In a game, questions or statements can also be played out as answers, which have an ironic or sarcastic background. In order to represent these correctly when selected by the Czar in the black cards, the representation in the black cards should be shown in quotation marks (or equivalent).

Possible scenarios of interpretation:
1. if a white card has no punctuation mark, it is displayed as before
2. if the white playing card has a (mistakenly) point, it is not shown.
3. if a "!" or "?" was specified as the last punctuation mark in the white card, the representation in the black card (when selected by the czar) is as is embedded with quotation marks -> "...?" or "...!")

**Comment by rfuehrer** [comment]
If it is a slight adjustment, I am happy to implement it - if it is what you want. If you don't have the time, please tell me in which files the necessary adjustments have to be made, then I will go deeper and try to prepare a PR for you. I haven't developed with Elm yet, but I could use a reference from "response.elm" in line 88/89 as a possible approach.

**Comment by Lattyware** [comment]
I'm not quite sure what you are saying here.

If a white card is intended to be displayed with quotes, that should be written into the card when it is created. I wouldn't want to automatically edit cards as it reduces choices for the person writing the card.

If you could provide a little more detail on or examples of the use case for this, and if this is something you are talking about happening generally, or in specific situations?

**Comment by rfuehrer** [comment]
We build up a set with colleagues in which answers appear as questions. These are sometimes formulations that appear as a question or statement. The white cards then end in the grammatically correct one with "!" or "?". 

Example (Regardless of the meaning;) ): 

"Spaghetti? Again?" (white card)
"The healing process began when I joined the support group for ____" (black card)
Addition: In German the placeholder is in the middle of the sentence "Der Heilungsprozess begann als ich der Support-Gruppe für ____ beigetreten bin".

In these examples it is a question of whether the result can be read more easily and emphasized correctly by the czar.

_The healing process began when I joined the support group for Spaghetti? Again?_ 
_The healing process began when I joined Spaghetti? Again? support group._

or 

_The healing process began when I joined the support group for "Spaghetti? Again?"._
_The healing process began when I joined "Spaghetti? Again?" support group._

The question mark might make the version with the quotation marks easier to read.

However, the request refers only to black cards where the fill-in field is in the middle of the sentence. In the example 

_What's under the desk? _____ _

the quotation marks would cause a different context. 

I've been rechecking our black and white cards. There are more constellations in which it gets even more complicated (another example with "Yes, but...") You could never do justice to these variations completely, because there can always be misunderstandings. The examples above do fit, but not every white card has to fit to a black card and maybe it is exactly the distinguishing feature for the winner selection by the tsar.

In that case I am not averse to closing the question and we rather reformulate our cards.

**Comment by Lattyware** [comment]
Sorry for not getting back to you on this.

I think in general this is too hard to implement generally: simpler rules like these tend to break the intent of some cards. In general I think preserving the original card is better, as users can generally make do even if it is a little off, but it is really annoying if you intentionally write a card one way and it gets modified in a way that doesn't work.

The best solution to this is, I think, in writing decks that mesh together well so any white card fits into any slot. Of course, that's not a trivial task, but it holds.

My instinct is if we wanted to try and do something more general, it would involve allowing decks to specify slot types, so you had something like noun slots and adjective slots, and each white card in such a deck would have to give a version for each type of slot. However, that comes back around to creating a lot of work for deck authors, so I'm not sure it would be worth implementing.

For now I'll close this, but please do feel free to explore the idea if you want.

**Comment by rfuehrer** [comment]
we have decided to formulate the cards accordingly and to have them verbally reformulated by the tsar. Our original approach is probably based on the fact that we are addressing the topic of "agility" and include nouns, verbs or even typical formulations or statements as white cards. Of course, depending on the black card, this only works to a limited extent.

After we have determined the complexity, our proposal might work, but it would differ greatly from the original and would not fit into the original pattern. We helped ourselves in our variant by including 80 black cards, 250 white cards and 50 blank cards in the game. This way we a) collect new impulses and b) can perhaps avoid the bumpy syntax (in which a strange sounding card can be reformulated by the player).

In so far I follow your suggestion to close the issue
